drivers/pwm/Kconfig.mcux

```diff
@@ -0,0 +1,125 @@
+# Kconfig.stm32 - STM32 PWM configuration options
+#
+#
+# Copyright (c) 2019 Linaro Limited.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig PWM_MCUX
+	bool "MCUX PWM driver"
+	depends on HAS_MCUX_PWM
+	help
+	  Enable mcux pwm driver.
+
+config PWM_MCUX_0
+	bool "NXP MCUX PWM 0 Output"
+	  depends on PWM_MCUX
```

Comment:
Wrap all the instance configs with a big `if PWM_MCUX...endif` instead of repeating this `depends on`.

Reply:
done